CC_ggplot2_cheatsheet.Rmd

```diff
@@ -0,0 +1,169 @@
+# ggplot2_cheatsheet
+
+Xiaorui Zhang
+## 1. Some Advanced Techs from ggplot2 
+
+### 1.1 Dataset
+```{r}
+library(ggplot2)
+library(dplyr)
+## [, 1]	mpg	Miles/(US) gallon
+## [, 2]	cyl	Number of cylinders
+## [, 3]	disp	Displacement (cu.in.)
+## [, 4]	hp	Gross horsepower
+## [, 5]	drat	Rear axle ratio
+## [, 6]	wt	Weight (1000 lbs)
+## [, 7]	qsec	1/4 mile time
+## [, 8]	vs	Engine (0 = V-shaped, 1 = straight)
+## [, 9]	am	Transmission (0 = automatic, 1 = manual)
+## [,10]	gear	Number of forward gears
+## [,11]	carb	Number of carburetors
+head(mtcars)
+```
+
+### 1.2 Facet
```

Comment:
Please stick to hashtags as opposed to using both hashtags and numbered chapters. When rendered, the hashtags will automatically be numbered as subsections.